node/resolvers/reviews.ts

```diff
@@ -6,6 +6,36 @@ declare var process: {
   }
 }
 
+/*This is a hack used to test the layout on motorolaus, but this should NEVER be used in
+practice because this is an extremely bad design choice that does not scale. The stores
+should configure bazaarvoice secondary ratings to have labels. */
+const parseSecondaryRatingsData = (secondaryRatingsData: any) => {
+  if (secondaryRatingsData.Label || secondaryRatingsData.Label != null) {
+    return secondaryRatingsData
+  } else {
```

Comment:
Remove unnecessary `else` clause
```suggestion
  }
```